apps/website/src/pages/docs/carousel.mdx

```diff
@@ -27,24 +25,22 @@ The currently shown slide can also be changed using the left/right arrow keys or
 
 ## Usage
```

Comment:
L22 can have links to the sub-components' headings. E.g. "dots" (Carousel.DotsList), "previous"/"next" buttons (Carousel.Navigation). This can help developers know how to implement the functionality we are mentioning in the description.

Reply:
Ohh, I actually meant the link should be on the component name.

```diff
- consisting of ["dots"](#dots-list) and ["previous"/"next"](#navigation) buttons, used for changing slides.
+ consisting of "dots" ([Carousel.DotsList](#dots-list)) and "previous"/"next" ([Carousel.Navigation](#navigation)) buttons, used for changing slides.
```